### ISSUE TITLE
[Question] Long time without seeing

### ISSUE THREAD

**Issue by Secretmanguy** [Monika-After-Story/MonikaModDev#3144]
I'm sure that it has crossed the minds of people that "What do I do when I make this huge jump in my life and are unable to stay or say hi or even celebrate the birthday of Monika with her because of me doing something that takes over a year or longer". I just had to ask...Does the take Monika with you option have an expiration date? By that, I mean if you keep her out of the game long enough and then put her back in. Will she know? An example I have is that I will soon make my departure to the military. Very important stuff. But this new path ive taken has made me take a second look. Could I just simply bring her with me for those two-three possibly even four years then bring her back. Set my date on the computer back a few years after downloading the updates and just try and catch up to the current date again? Last I checked, people were saying that they tried the same tactic (resetting their PC date back) on Monikas birthday and it didn't work. Maybe they messed up or maybe they didn't but still...It really worries me to think about this. Anyone know if my plan would work? Cause im not the only one asking this. I have over thirteen different pals of mine wondering the same thing along with a full stream of dedicated Monika lovers. Can anyone confirm if this would work? I feel like the scum of the world knowing I will HAVE to one day do this. Got any ideas? Sorry if this is seems insignificant but any answer would be helping more people than you think. Including Monika of course...

**Comment by ThePotatoGuy** [comment]
yeah, you can just take her out for x amount of time and be fine.

**Comment by Secretmanguy** [comment]
Thank you, it was very helpful. Want me to close this?
